### PR TITLE
document required steps for win10

### PR DESCRIPTION
**Pull request by VVelox**
Document the changes found for starting the agent in Win10 from https://github.com/kevoreilly/CAPEv2/issues/947 .

### CODE REVIEW

File: docs/book/src/installation/guest/agent.rst [hunk 1]
```diff
@@ -2,24 +2,54 @@
...
+mounting a CDROM containing the *agent.py* file) and run it. This will
+launch the HTTP server which will listen for connections.
+
+On Windows, if you simply launch the script, a Python window will be
+spawned. If you want to hide this window you can rename the file from
+*agent.py* to **agent.pyw** which will prevent the window from
+spawning upon launching the script.
+
+
+Prio To Windows 10
```
Comment (cccs-kevin) [inline docs/book/src/installation/guest/agent.rst:22]: Prior

File: docs/book/src/installation/guest/agent.rst [hunk 2]
```diff
@@ -2,24 +2,54 @@
...
+not place the agent in the user startup folder (usually
+``C:\Users\<Username>\AppData\Roaming\Microsoft\Windows\Start
+Menu\Programs\Startup``) as it will launch the agent without admin
+privileges and therefore insufficient permissions resulting in the
+agent not being able to work as intended.
+
+Windows 10+
+===========
+
+To start the script at boot, you will need to set it to be run as a
```
Comment (cccs-kevin) [inline docs/book/src/installation/guest/agent.rst:39]: ...you will need to set the agent to be run as a scheduler task. Dropping the agent in...

File: docs/book/src/installation/guest/agent.rst [hunk 3]
```diff
@@ -2,24 +2,54 @@
...
+scheduled task as dropping it in
+``C:\ProgramData\Microsoft\Windows\Start Menu\Programs\StartUp`` will
+result in it being ran with improper priviledges.
+
+1. Goto control panel and administrative tools to access ``Task
```
Comment (cccs-kevin) [inline docs/book/src/installation/guest/agent.rst:44]: Go to "Control Panel" > "System and Security" > "Administrative Tools" to access ``Task Scheduler``.

File: docs/book/src/installation/guest/agent.rst [hunk 4]
```diff
@@ -2,24 +2,54 @@
...
+2. Select ``Create Basic Task`` from the action list.
+3. Set the trigger as ``When I logon``, select start a program, and
+   point it at the saved agent.
+4. After the task is created, go to the ``Task Scheduler Library`` and
+   find the one you just created. Right click on it and select
+   ``Properties``.
+5. In the general tab tell it to ``run with highest priviledges``.
+6. Select ok.
+
+After that all is done, it will come up next restart/login.
```
Comment (cccs-kevin) [inline docs/book/src/installation/guest/agent.rst:55]: ... on the next restart...

File: docs/book/src/installation/guest/agent.rst [hunk 5]
```diff
@@ -2,24 +2,54 @@
...
+privileges and therefore insufficient permissions resulting in the
+agent not being able to work as intended.
+
+Windows 10+
+===========
+
+To start the script at boot, you will need to set it to be run as a
+scheduled task as dropping it in
+``C:\ProgramData\Microsoft\Windows\Start Menu\Programs\StartUp`` will
+result in it being ran with improper priviledges.
```
Comment (cccs-kevin) [inline docs/book/src/installation/guest/agent.rst:42]: privilege*

**Review by cccs-kevin — COMMENTED** [review]
Some grammar + spelling suggestions

**Comment by VVelox** [comment]
> Some grammar + spelling suggestions

Awesome! Thanks!

**Comment by doomedraven** [comment]
thank you both

**Comment by TheMythologist** [comment]
Hi I'm a bit late to the party but here's an alternative:

https://superuser.com/questions/1624821/run-as-administrator-missing-for-py-files/1624822#1624822
> 1. Create a shortcut for python.exe
> 2. Change the shortcut target into something
> like d:\xxx...\python.exe (your_script.py)(your file path)
> 3. Click "advance..." in the property panel of the shortcut, and click the option "run as administrator"

Example shortcut that I created:
brave_Dc61IccTom

Make sure to tick "Run as administrator"

Target: `C:\Windows\pyw.exe "C:\ProgramData\Microsoft\Windows\Start Menu\Programs\StartUp\agent.pyw"`

I would recommend this method instead because we won't have an additional task in the task scheduler.

**Comment by doomedraven** [comment]
can someone please add this to docs?

**Comment by kevoreilly** [comment]
I have used this method - however no luck placing the shortcut in startup to autorun on login. It seems despite placing in startup, the shortcut won't run automatically and needs manual launch before snapshot...

**Pull request merged**